cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/ClusterClassVariableDisplay.tsx

```diff
@@ -186,33 +192,37 @@ export function createFormSchema(cc: ClusterClassDefinition | undefined) {
         return undefined
     }
     const schemaObject = allClusterClassVariables(cc).reduce<any>((accumulator, ccVar) => (
-        {...accumulator, [ccVar.name]: createYupObjectForClusterClassVariable(ccVar)}
+        { ...accumulator, [ccVar.name]: createYupObjectForClusterClassVariable(ccVar) }
     ), {})
     return yup.object(schemaObject);
 }
 
 function createYupObjectForClusterClassVariable(ccVar: ClusterClassVariable) {
     let yuppy
     switch (ccVar.valueType) {
-        case ClusterClassVariableType.STRING:
-            yuppy = yup.string().nullable()
-            break
-        case ClusterClassVariableType.BOOLEAN:
-            yuppy = yup.boolean().nullable()
-            break
-        case ClusterClassVariableType.CIDR:
-            yuppy = yup.string().test('', 'Please enter a CIDR', value => (!ccVar.required && !value) || isValidCidr(value) )
-            break
-        case ClusterClassVariableType.IP:
-            yuppy = yup.string().test('', 'Please enter a valid ip or fqdn',
-                    value => (!ccVar.required && !value) || isValidFqdn(value) || isValidIp(value))
-            break
-        case ClusterClassVariableType.IP_LIST:
-            yuppy = yup.string().test('', 'Please enter a comma-separated list of valid ip or fqdn values',
-                value => isValidCommaSeparatedIpOrFqdn(value))
-            break
-        default:
-            yuppy = yup.string().nullable()
+    case ClusterClassVariableType.STRING:
+        yuppy = yup.string().nullable()
+        break
+    case ClusterClassVariableType.STRING_K8S_COMPLIANT:
+        yuppy = yup.string().test('', 'Please enter a string containing only lower-case letters and hyphens',
```

Comment:
Minor detail to follow up on - I believe most K8s object names are alphanumeric, but if our CLI is validating in a different way, please disregard.

https://kubernetes.io/docs/concepts/overview/working-with-objects/names/